ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -1369,7 +1405,25 @@ void instr_execute_BLEND(in instr_t instr, in uvec3 regs, in mat2 dUV, in bsdf_d
 void instr_execute(in instr_t instr, in uvec3 regs, in mat2 dUV, in vec3 L)
 {
 	bsdf_data_t bsdf_data = fetchBSDFDataForInstr(instr);
-	switch (instr.x & INSTR_OPCODE_MASK)
+	uint opcode = instr_getOpcode(instr);
+	uint ndf = instr_getNDF(instr);
+	bool twosided = instr_getTwosided(instr);
+	
+	//if (twosided)
+	//flip normal
```

Comment:
if twosided and `NdotV<0.0` you can flip sign of `N` and any `Ndot` var, plus anything to do with the bitangent so `B` and `Bdot`

oooor, you could take it into account when carrying out the `OP_SET_GEOM_NORMAL` and have the twosided bit only be used for that instruction (and freeing up that bit for other instructions)... yes in this case if `twosided` changes you need to emit an extra `OP_SET_GEOM_NORMAL`

Reply:
let me know if tackled in new implementation.

Reply:
?

Reply:
done?

Reply:
yes, ofc